packages/common/mui/src/lib/data-table-cells/multi-value-icon-cell/multi-value-icon-cell.tsx

```diff
@@ -12,21 +12,21 @@ import {
 } from '../../typography/typography';
 import { Network } from '@notional-finance/util';
 
-export interface MultiValueIconCellProps {
-  cell: {
-    value: {
-      symbol: string;
-      label: string;
-      labelIsNegative?: boolean;
-      symbolSize?: string;
-      caption?: string;
-      inlineIcons?: boolean;
-      network?: Network;
-    };
-  };
-  row: { original };
-  column: { id; textAlign };
-}
+// export interface MultiValueIconCellProps {
```

Comment:
I personally find these type definitions for all the different cells useful. When I'm using it, it's hard to tell if I'm putting the right values and what type each value needs to be without having to read through all the code.